client/packages/common/src/intl/locales/en/common.json

```diff
@@ -78,6 +78,7 @@
   "error.date_maxDate": "Date value is too large",
   "error.date_minDate": "Date value is too low",
   "error.general-query-error": "There was an error communicating with the server 😢",
+  "error.invalid-credentials": "Unable to fetch user details. Please re-login to update user details.",
```

Comment:
Honestly would prefer to have the user log out straight away if updating fails since it's easy to ignore a message... but shouldn't happen in production often... more likely to happen in development with constantly server restarts and branch changes